README.md

```diff
@@ -9,15 +9,27 @@ wherever it is possible. Unfortunately legacy builds `coreboot-4.0.x` require
 toolchain from pre-coreboot-sdk era that's why we created
 `pce-fw-builder-legacy`.
 
+Keep in mind that this tool works only for coreboot releases not older than
+v4.6.9 and v4.0.17. For older releases use the procedure described in
```

Comment:
@krystian-hebel Lets consider v4.6.9 as a transient release that is not fully supported by this tool. Mention that the earliest supported 4.6.x version is v4.6.10

Reply:
@miczyg1 OK. Should I change checks in scripts too?

Reply:
@krystian-hebel please do. Also You should update the apu2-documentation repository, as it contains info about pce-fw-builder too.